git-imerge

```diff
@@ -3629,9 +3629,22 @@ def cmd_diagram(parser, options):
 
 
 def cmd_reparent(parser, options):
+    if len(options.parents) == 0:
+        parser.error('Parent is not specified, please specify at least one parent.')
+    sys.stdout.write('Reparenting from HEAD to %s...\n' % options.commit_to_reparent)
```

Comment:
It is not appropriate to emit progress messages to stdout. They should go to stderr so that the stdout can be piped to another program.